src/components/ColorPicker/style.css

```
@@ -0,0 +1,124 @@
+.popover :global(.ant-popover-content) {
+    width: 215px;
+    box-shadow: 0px 3px 6px #000000;
+}
+
+.popover :global(.ant-popover-inner-content) {
+    padding: 10px 8px;
+    display: flex;
+    justify-content: center;
+}
+
+.container {
+    padding-left: 4px;
+}
+
+.container .selection-display {
+    display: flex;
+}
+
+.container .old-color,
+.container .new-color {
+    height: 30px;
+    width: 60px;
+}
+
+.container .old-color-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    font-size: 9px;
+    color: var(--white-two);
+    gap: 4px;
+}
+
+.container .hex-input {
+    height: 30px;
+    width: 68px;
+    background-color: transparent;
+    border: 1px solid var(--white-two);
+    font-size: 14px;
+    text-transform: uppercase;
+    padding-left: 6px;
+    padding-top: 4px;
+}
+
+
+.container .new-color {
+    margin-right: 8px;
+}
+
+.container .agent-colors-swatches {
+    padding-top: 6px;
+    padding-bottom: 12px;
+}
+
+.container .colors {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 7px;
+}
+
+.container .swatch {
+    height: 15.5px;
+    width: 15.5px;
+    cursor: pointer;
+    border: none;
+}
+
+.container .tooltip {
+    box-shadow: 0px 3px 4px #000000;
+    height: 20px !important;
+}
+
+.tooltip :global(.ant-tooltip-inner) {
+    font-size: 9px;
+    letter-spacing: 0px;
+    min-height: 0px;
+    padding: 5px 10px 16px 10px;
+    text-transform: uppercase;
+    height: 20px !important;
+}
+
+.container .recent-color-text {
+    text-align: left;
+    font-size: 12px;
+    color: var(--white-two);
+    margin-bottom: 2px;
+    font-weight: 300;
+}
+
+.container :global(.react-colorful__saturation-pointer) {
+    width: 0px;
+    height: 0px;
+    border-radius: 0px;
+    border: none;
+}
+.container :global(.react-colorful__hue-pointer) {
+    width: 5px;
+    height: 16px;
+    border: 1px solid var(--white-two);
+    border-radius: 0px;
+}
+
+.container :global(.react-colorful__pointer-fill) {
+    background-color: black !important;
+}
+
+.container :global(.react-colorful) {
+    gap: 7px;
+    padding-bottom: 8px;
+    height: auto;
+}
+
+.container :global(.react-colorful__saturation) {
+    border-radius: 0px;
+    height: 115px;
+    width: 196px;
+}
+
+.container :global(.react-colorful__hue) {
+    border-radius: 0px;
+    width: 196px;
+    height: 12px;
+}
```

```suggestion
}

```
Nit: Whitespace at end of file (This happens in a few places in this change. I have VSCode set up to automatically format files when I save, which usually catches this? Not sure if you already have that set up but it changed my life when I found out about it)